docs/en/setup/backend/dynamic-config-etcd.md

```diff
@@ -7,11 +7,54 @@ configuration:
   selector: ${SW_CONFIGURATION:etcd}
   etcd:
     period: ${SW_CONFIG_ETCD_PERIOD:60} # Unit seconds, sync period. Default fetch every 60 seconds.
-    endpoints: ${SW_CONFIG_ETCD_ENDPOINTS:localhost:2379}
+    endpoints: ${SW_CONFIG_ETCD_ENDPOINTS:http://localhost:2379}
     namespace: ${SW_CONFIG_ETCD_NAMESPACE:/skywalking}
     authentication: ${SW_CONFIG_ETCD_AUTHENTICATION:false}
     user: ${SW_CONFIG_ETCD_USER:}
     password: ${SW_CONFIG_ETCD_password:}
 ```
 
-**NOTE**: Only the v3 protocol is supported since 8.7.0.
\ No newline at end of file
+**NOTE**: Only the v3 protocol is supported since 8.7.0.
+
+## Config Storage
+### Single Config
+Single configs in etcd are key/value pairs:
+
+| Key | Value |
+|-----|-----|
+| {nameSpace}/configKey | configVaule |
```

Comment:
I just noticed this, please check all documents.

```suggestion
| {namespace}/configKey | configVaule |
```

`namespace` is one word.